### DISCUSSION TITLE
Unable to login to admin account after setting up auto-direct to first login method

### DISCUSSION THREAD

**Discussion by HikigayaHachiman0428** [requarks/wiki#5536] [Help / Questions] [answered]
I have enabled the feature of "auto-directing to first login method" in the login setup of my wiki, and since first login method doesn't allow an admin login, I wonder if there is a way I can fix this issue and login the wiki as admin. Thanks a lot!

**Comment by HikigayaHachiman0428** [comment, marked as answer]
solved by using login?all=1

**Reply by ganesh112700** [reply]
How to enable the "auto-directing to first login method"

**Reply by NGPixel** [reply]
@ganesh112700 Under admin > security.

**Comment by ganesh112700** [comment]
In wiki.js without login itself, my homepage content is coming, I installed the wiki.js in ubuntu18.04 machine, and its working as a service, but the issue is without user login itself users can see my wiki.js content, I need a solution for if users try to access the wiki.js the login page should come first,so kindly give a solution for this.

my-email-ID: [ganeshaw11@gmail.com](mailto:ganeshaw11@gmail.com)
